frontend/src/App.js

```diff
@@ -28,18 +29,19 @@ function App() {
             <Router>
                 <Routes>
                     <Route exact path="/" element={<Navigate replace to="/signup" />}></Route>
-                    <Route path="/signup" element={<SignupPage/>}/>
+                    <Route path="/signup" element={<SignupPage />} />
                     <Route path="/login" element={<LoginPage />} />
                     <Route path="/logout" element={<LogoutPage />} />
                     {/* Placeholder home for matching-service */}
                     <Route path="/" element={<WithNavBar />}>
                         <Route path="home" element={<Home />} />
-                        <Route path="changePassword" element={<ChangePasswordPage/>}/>
-                        <Route path="selectquestiondifficulty" element={<SelectionView/>}></Route>
-                        <Route path="findingmatch" element={<CountdownView/>}></Route>
-                        <Route path="roompage" element={<RoomPage/>}></Route>
+                        <Route path="changePassword" element={<ChangePasswordPage />} />
+                        <Route path="deleteAccount" element={<DeleteAccountPage />} />

```

Comment:
`deleteAccount` page should not be used with `WithNavBar`. This will cause navbar to appear on the resulting delete page, which is undesirable. You should move this to the `routes` outside of this block of pages with `WithNavBar`

![image](https://user-images.githubusercontent.com/65227564/195849707-fd2cf3a1-b015-48ab-9950-7ff17295848b.png)


Discussion: Perhaps we could think of a more elegant UI flow for showing that the account has been deleted.

Reply:
I agree. Removed in latest commit.